holidays/countries/somalia.py

```diff
@@ -0,0 +1,79 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from holidays.groups import InternationalHolidays, IslamicHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Somalia(HolidayBase, InternationalHolidays, IslamicHolidays):
+    """Somalia holidays.
+
+    References:
+        * <https://usa.mfa.gov.so/about-somalia/>
+        * <https://web.archive.org/web/20220607190753/https://aip.scaa.gov.so/eAIP/HC-GEN-2.1-en-GB.html>
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Somalia>
```

Comment:
```suggestion
        * <https://en.wikipedia.org/wiki/Public_holidays_in_Somalia>
        * <https://web.archive.org/web/20250522034818/https://usa.mfa.gov.so/about-somalia/>
```
I see nothing valuable in the third source.